spec/models/ext_management_system_spec.rb

```diff
@@ -725,33 +725,36 @@
     it "queues up destroy" do
       described_class.destroy_queue([ems.id, ems2.id])
 
-      expect(MiqQueue.count).to eq(2)
-      expect(MiqQueue.pluck(:instance_id)).to match_array([ems.id, ems2.id])
+      expect(MiqQueue.where(:method_name => "orchestrate_destroy").count).to eq(2)
+      expect(MiqQueue.where(:method_name => "orchestrate_destroy").pluck(:instance_id)).to match_array([ems.id, ems2.id])
     end
   end
 
   context "#destroy_queue" do
+    before       { Zone.seed }
     let(:ems)    { FactoryBot.create(:ext_management_system, :zone => zone) }
     let(:server) { EvmSpecHelper.local_miq_server }
     let(:zone)   { server.zone }
 
     it "returns a task" do
       task_id = ems.destroy_queue
 
-      deliver_queue_message
+      deliver_queue_message # Deliver the `#pause!` queue item
+      deliver_queue_message # Deliver the `#destroy` queue item
```

Comment:
Should this say (here and elsewhere below?)

```suggestion
      deliver_queue_message # Deliver the `#orchestrate_destroy` queue item
```

Reply:
:+1: yeah, updated